src/components/CreateProposal/CreateProposal.scss

```diff
@@ -64,8 +64,16 @@
   		border-radius: 3px;
   		font-size: $subheader;
   		padding: 8px;
-      font-family: $body-font;
-  	}
+			font-family: $body-font;
```

Comment:
man the whitespace in this doc is wack, ideally you should stick with the current system though

Reply:
Seems like my editor messed it up somehow...